worker/include/Utils.hpp

```diff
@@ -23,6 +23,18 @@ namespace Utils
 
 		static void GetAddressInfo(const struct sockaddr* addr, int& family, std::string& ip, uint16_t& port);
 
+		static size_t GetAddressLen(const struct sockaddr* addr)
+		{
+			if (addr->sa_family == AF_INET)
+			{
+				return sizeof(struct sockaddr_in);
+			}
+			else
+			{
+				return sizeof(struct sockaddr_in6);
+			}
```

Comment:
Please be explicit and check AF_INET6 since there are other sa_family values and we must abort or reject in thoser cases (as we do in other utils).